### PR TITLE
[Gideon]-Sage-Designing a User-Friendly Celo DApp: A Beginner's Guide to UI/UX

### PR DESCRIPTION
**Pull request by GideonNut**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

**Review by Ishanpathak1 — COMMENTED** [review]
Please make the changes and let me know

**Review by Ishanpathak1 — COMMENTED** [review]
Made the changes

File: blog/02-16-2023-Designing a user-friendly-dApp-A-beginner's-guide-to-UI/Designing-a-user-friendly-celo-dapp-a-beginners-guide-to-uiux.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
+---
+title: Designing a User-Friendly Celo DApp- A Beginners Guide to UI/UX
+description: In this tutorial, we will teach interested persons how to create wireframes for their own web projects, using the Celo website as an example.
+authors:
+- name: Gideon Nutekpor Yaw
+  title: Celo member , Celo foundation
+  url: https://github.com/GideonNut
+  image_url: https://github.com/GideonNut.png
+tags: [celo, celosage, beginner, dApp, celosage, valora]
```
[the remark this answers is not in the document]
Comment (GideonNut) [reply blog/02-16-2023-Designing a user-friendly-dApp-A-beginner's-guide-to-UI/Designing-a-user-friendly-celo-dapp-a-beginners-guide-to-uiux.md]: done

**Review by developerkunal — APPROVED** [review]
Approved @Ishanpathak1

**Pull request merged**